app/eq.py

```diff
@@ -128,6 +130,29 @@ async def build(self):
 
         return self._payload
 
+    @staticmethod
+    def encrypt_response_id(questionnaire_id: str):
+        """
+        Encrypts a response_id using the following format: <response_id><first 16 characters of sha256 signature>.
+        Utilises EQ_SALT environment property as salt.
+        Implements the HMAC algorithm as described by RFC 2104.
```

Comment:
I think this is a plain SHA256 rather than an HMAC , so I think this line can be removed. It is misleading.